### ISSUE TITLE
Boxes overlapping when going from small window to maximized window if media query resizes fonts

### ISSUE THREAD

**Issue by streppuiu** [desandro/masonry#822]
Hi David, I am having the issue  stated in the title with the backend of my site.

To reproduce it I am resizing the window to be small (200px maybe) and then maximize the window directly to 1080p. I noticed that this only happens when my media queries resize the fonts. If I remove font resizing querries all resize just fine.
The same happens if I do the reverse - going from maximized window to small sized window - but in this case the box is not overlapping but too much space remains between them.
In both cases my queries are resizing the fonts.
If then I resize the window smoothly a little, the boxes will go to their right place without issue.

I am using jquery 1.4.3 but I also tried with 1.12.0 and had the same issue. Tried with Firefox 44.0.1 and Chrome 48.0.2564.109 m (64-bit) , the issue reproduced in both browsers.

Here is the way I am initializing masonry:
`
var $MASONRYGRID;
$(window).load(function() {
    window.setTimeout(function(){
        $MASONRYGRID = $('.stBox .fields').masonry({
            columnWidth: '.grid-sizer',
            percentPosition: true,
            gutter: '.gutter-sizer'
        });
        $('.stBox > .loading').css('display', 'none');
        $('.stBox > *').css('visibility', 'visible');
        if(funcExists("afterMasonryLoad")){
            afterMasonryLoad();
        }
    }, 500);
});
`

I am providing a few pics and the page I am experiencing this issue in a zip file, it can be opened in the browser directly. Styles are in the "main.css" file.

I know this is not a reduced test case and I will try to provide one if this does not help.

Here are the links to  the files:

[chrome1](https://www.dropbox.com/s/1jfzptddtnaoxb9/Chrome_after_maximize_window.png?dl=0)
[firefox1](https://www.dropbox.com/s/0hzpp64gcrf26ph/firefox_after_maximize_window.png?dl=0)
[firefox2](https://www.dropbox.com/s/zqgc1pgrfecr74t/Firefox_small_width.png?dl=0)
[test_page](https://www.dropbox.com/s/gnduit7lamhqb2m/test_page.zip?dl=0)

Thanks,
Ioan

**Comment by desandro** [comment]
Sorry to hear you're having trouble with Masonry. I appreciate you providing the files, but I really need a **reduced test case** on a live URL in order to debug.  See [Submitting Issues in the contributing guidelines](https://github.com/desandro/masonry/blob/master/CONTRIBUTING.mdown#submitting-issues).

**Comment by streppuiu** [comment]
I understand. It's pretty difficult for me to strip everything from the web page and try to recreate the issue without all of the other things. I could however upload the page to a web server if you think it would help.

Anyway, if you have any idea of what could cause this issue, if it could be related to the size of the text being changed and what could be done in that case I would appreciate your input.

Thanks again.

**Comment by streppuiu** [comment]
I figured this out finally and I wanted to let you know what the problem was. I had a few buttons that had css transition for all their properties and their size depended on the size of fonts used. So when the page went to full size from small size, font size increased (because of a media query) and the size of the buttons increased too but because the end of transition was happenning after masonry fitted everything some of the boxes in which the buttons were placed growed and overlapped those below. I restrincted the transition to their color and now all works fine.
